torchvision/transforms/v2/functional/_geometry.py

```diff
@@ -1401,18 +1373,7 @@ def _crop_bounding_boxes_dispatch(
 
 @_register_kernel_internal(crop, tv_tensors.Mask)
 def crop_mask(mask: torch.Tensor, top: int, left: int, height: int, width: int) -> torch.Tensor:
-    if mask.ndim < 3:
-        mask = mask.unsqueeze(0)
-        needs_squeeze = True
-    else:
-        needs_squeeze = False
-
-    output = crop_image(mask, top, left, height, width)
-
-    if needs_squeeze:
-        output = output.squeeze(0)
-
-    return output
+    return crop_image(mask, top, left, height, width)
```

Comment:
Since the introduction of the (un-)squeeze logic here, we seem to have refactored the image kernel. I was able to remove that without touching the image kernel.